calculator/src/components/Form.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Activities from './Activities';
+
+export default class Form extends React.Component {
+  state = {
+    sleep: '',
+    exercise: '',
+    meals: '',
+    reading: '',
+    other: ''
+  }
+
+  handleOnChange = e => {
+    this.setState({
+      [e.target.name]: e.target.value
+    })
+  }
+
+  handleSubmit = ev => {
+    ev.preventDefault();
```

Comment:
You are already tracking state of all the inputs in your `handleOnChange` so you could probably get rid of `handleSubmit` altogether (or at least get rid of `this.setState` inside `handleSubmit` and perform other tasks in it, like showing a message to the user).